### PR TITLE
feat(vulnerability_summary_page): Vulnerability summary page integration

### PR DESCRIPTION
**Pull request by deo002**

### CODE REVIEW

File: src/app/[locale]/vulnerabilities/detail/[id]/components/VulnerabilityDetailsTab.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                )
+                if (response.status === HttpStatus.UNAUTHORIZED) {
+                    return signOut()
+                } else if (response.status !== HttpStatus.OK) {
+                    return notFound()
+                }
+
+                const data = await response.json()
+
+                setSummaryData({
```
Comment (hoangnt2) [inline src/app/[locale]/vulnerabilities/detail/[id]/components/VulnerabilityDetailsTab.tsx:58]: You don't need to do this. You just need to use setSummaryData(data) and [conditional rendering](https://react.dev/learn/conditional-rendering) to render data in jsx.

**Review by hoangnt2 — CHANGES_REQUESTED** [review]

**Comment by hoangnt2** [comment]
@deo002, and please add more css to make the GUI like the old one. You can copy them from my component detail page
image

File: src/app/[locale]/vulnerabilities/detail/[id]/components/VulnerabilityDetailsTab.tsx (hunk 1 above)
Comment (deo002) [reply src/app/[locale]/vulnerabilities/detail/[id]/components/VulnerabilityDetailsTab.tsx]: Done

**Comment by deo002** [comment]
> @deo002, and please add more css to make the GUI like the old one. You can copy them from my component detail page image

Done! It looks beautiful now!

File: src/app/[locale]/vulnerabilities/detail/[id]/components/Summary.tsx [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+                        <td>{t('External Id')}:</td>
+                        <td>{summaryData.externalId && summaryData.externalId}</td>
+                    </tr>
+                    <tr>
+                        <td>{t('Publish Date')}:</td>
+                        <td>{summaryData.publishDate && summaryData.publishDate}</td>
+                    </tr>
+                    <tr>
+                        <td>{t('Last update')}:</td>
+                        <td>{summaryData.lastUpdateDate && summaryData.lastUpdateDate}</td>
```
Comment (hoangnt2) [inline src/app/[locale]/vulnerabilities/detail/[id]/components/Summary.tsx:55]: This should be **lastExternalUpdate** instead of **lastUpdateDate**

**Review by hoangnt2 — CHANGES_REQUESTED** [review]

File: src/app/[locale]/vulnerabilities/detail/[id]/components/Summary.tsx (hunk 2 above)
Comment (deo002) [reply src/app/[locale]/vulnerabilities/detail/[id]/components/Summary.tsx]: Done

**Review by hoangnt2 — APPROVED** [review]
Change look good

**Pull request merged**